mariamihaelafainarea/assignment3/Assignment3

```diff
@@ -0,0 +1,192 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package assignment3;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Set;
+import java.util.TreeSet;
+
+/**
+ *
+ * @author maria
+ */
+public class Assignment3 {
+    //dictionary of the words and the grid
+    private char[][] grid;
+    private MyDictionary dictionary;
+    private int n,m;
+
+    public MyDictionary getDictionary() {
+        return dictionary;
+    }
+
+    public void setDictionary(MyDictionary dictionary) {
+        this.dictionary = dictionary;
+    }
+
+    public char[][] getGrid() {
+        return grid;
+    }
+
+    public void setGrid(char[][] grid) {
+        this.grid = grid;
+    }
+    
+    public void initializare(char[][] grid, MyDictionary dictionary){
+        //here you receive the structures from somewhere
+        this.grid = grid;
+        this.dictionary = dictionary;
+        n = grid.length;
+        m = grid[0].length;
+    }
+    
+    //i need many DFS to start from any position
+    //i am using a set because
+    //it can appear duplicate words if a cell has the same character as
+    //another one
+    public Set<String> manyDFSs() {
+        //ManyDFS has the role to start from any point of the
```

Comment:
These two comments seem to repeat each other :) Maybe just try and have one comment before a method (a so-called java-doc) that briefly explains what the method does. I don't think you really need to explain why you are using set: hopefully it will be more or less obvious from the context :)

Reply:
okey